assets/src/js/product-editor.js

```diff
@@ -1002,49 +1002,6 @@
             return false;
         });
 
-        function dokan_show_earning_suggestion( callback ) {
-            let commission = $('span.vendor-earning').attr( 'data-commission' );
-            let product_id = $( 'span.vendor-earning' ).attr( 'data-product-id' );
-            let product_price = $( 'input.dokan-product-regular-price' ).val();
-            let sale_price = $( 'input.dokan-product-sales-price' ).val();
-            let earning_suggestion = $('.simple-product span.vendor-price');
-
-            earning_suggestion.html( dokan.i18n_calculating );
-
-            $.get( dokan.ajaxurl, {
-                action: 'get_vendor_earning',
```

Comment:
Also need to remove the ajax action hook and callback in the following php file.

* [includes/Ajax.php #L56](https://github.com/weDevsOfficial/dokan/blob/develop/includes/Ajax.php#L156)